<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FindBar</class>
 <widget class="QWidget" name="FindBar">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>250</width>
    <height>54</height>
   </rect>
  </property>
  <property name="autoFillBackground">
   <bool>true</bool>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>4</number>
   </property>
   <property name="topMargin">
    <number>4</number>
   </property>
   <property name="rightMargin">
    <number>4</number>
   </property>
   <property name="bottomMargin">
    <number>4</number>
   </property>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLineEdit" name="searchBox">
       <property name="placeholderText">
        <string>Find in program</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="previousButton">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>&lt;</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="nextButton">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>&gt;</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="closeButton">
       <property name="text">
        <string>X</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QLabel" name="matchesLabel"/>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
